Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,12 @@
 import React from 'react';
 import './App.css';
-
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import ContainerButtons from './components/ButtonComponents/ContainerButtons';
 const App = () => {
```

Comment:
_Usually_ your `App.js` is going to be a class component. And with this calculator, it almost has to be, unless you can find a way around using state here 🤔